test/AutoRest.TestServerLowLevel.Tests/dpg-version.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using Azure;
+using AutoRest.TestServer.Tests.Infrastructure;
+using TypeSpec.Versioning.Latest;
+using TypeSpec.Versioning.Latest.Models;
+using NUnit.Framework;
+using System.Linq;
+using System.Collections.Generic;
+using Resource = TypeSpec.Versioning.Latest.Models.Resource;
+using System.Threading;
+
+namespace AutoRest.TestServer.Tests
+{
+    public class DpgVersionTest : TestServerLowLevelTestBase
```

Comment:
If you are not calling test server, there is no need to inherit TestServerLowLevelTestBase

Reply:
removed the base class